library/r-00001-the-interface-will-support-gui-web-access-to-end-users.md

```diff
@@ -1,3 +1,23 @@
 ---
-title: The interface will support GUI web access to end users
+layout: default
+title: "R00001 The interface will support GUI web access to end users &#x2705; "
+nav_order: 3
+parent: Requirements
+has_children: false
 ---
+
+# R00001 The interface will support GUI web access to end users
```

Comment:
I _think_ this can be removed as it will be automatically generated in the final web site.

Reply:
Yes, requirements pages just need the yaml header